pom.xml

```diff
@@ -306,17 +308,13 @@
         <classifier>runtime</classifier>
       </dependency>
       <dependency>
+        <!-- Do not upgrade, leads to test failures, needs analysis -->
         <groupId>org.junit</groupId>
         <artifactId>junit-bom</artifactId>
         <version>5.9.3</version>
         <type>pom</type>
         <scope>import</scope>
       </dependency>
-      <dependency>
-        <groupId>org.codehaus.plexus</groupId>
-        <artifactId>plexus-xml</artifactId>
-        <version>3.0.0</version>
-      </dependency>
```

Comment:
I hope is is not used by any transitive dependencies 😄

Reply:
I don't think so. It is brand new. Parent sets to 3.0.0 already.